src/core/FeaturedLink.tsx

```diff
@@ -54,9 +55,17 @@ const FeaturedLink = ({
   return (
     <a
       {...(onClick ? {} : { href: url })}
-      className={`font-sans font-bold block ${disabled ? "text-gui-unavailable pointer-events-none" : "text-gui-default hover:text-gui-hover focus:text-gui-focus focus:outline-gui-focus"} group/featured-link ui-${textSize} ${
-        flush ? "" : "py-8"
-      } ${additionalCSS}`}
+      className={cn(
+        "font-sans font-bold block group/featured-link",
+        { "text-gui-unavailable pointer-events-none": disabled },
```

Comment:
`clsx` is pretty open ended - we could do this with the "class as key, condition as value" object convention as is here, or have this as a ternary, or whatever really. I went with the former here to cut the line length down. You _could_ have all of this as one object, and have the always-on classes with `true` as the value, but that seems verbose to me here.

I like it's flexibility though. If I were to make a fix here it would be to update the template literal for a string literal on l66 and remove the object wrapping for l67